src/go/rpk/pkg/cli/cmd/root.go

```diff
@@ -70,7 +70,7 @@ func Execute() {
 	rootCmd.AddCommand(NewGenerateCommand(mgr))
 	rootCmd.AddCommand(NewVersionCommand())
 	rootCmd.AddCommand(NewApiCommand(fs, mgr))
-	rootCmd.AddCommand(NewWasmCommand(fs))
+	rootCmd.AddCommand(NewWasmCommand(fs, mgr))
```

Comment:
i would change the commit to be more understandable: "add 'deploy' and 'remove' to rpks global list of available commands"